task1/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "bucket_name" {
+    type        = string
+    description = "Value of the Name tag for the S3 Bucket"
+    default     = "First TF Bucket"
```

Comment:
the default value should be a valid input. in this case, you can not have space " " in bucket name.